schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
-    name varchar(100)
+CREATE TABLE animals(
+   id INT PRIMARY KEY GENERATED ALWAYS AS IDENTITY NOT NULL,
+   name           VARCHAR    NOT NULL,
```

Comment:
- [OPTIONAL] Please it will be nice to specify the length of the name field, something like so: 
```
name varchar(255)
```